calcom/cal.com
# fix: not able to reschedule an event/meeting when new time slot overlaps with old selected time slot

## What does this PR do?

Fix not able to reschedule an event/meeting when new time slot overlaps with old selected time slot

**Environment**: Staging(main branch) / Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Create a new event type of length 60 minutes
- Use `Time-slot intervals` as 15 minutes and save the event
- Now open the event link and schedule a booking
- Now try to re-schedule the event and verify the time slot near to the previously selected time slot is available

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
